be/src/exec/vectorized/file_scan_node.h

```diff
@@ -59,10 +59,10 @@ class FileScanNode final : public ScanNode {
     }
 
     // Create scanners to do scan job
-    Status _start_scanners();
+    Status _start_scanners(bool skip_headline);
```

Comment:
should move skip_headline to TBrokerScanRangeParams, you can refer to the implementation of multi_column_separator